templates/components/plugin_update_modal.html.twig

```diff
@@ -0,0 +1,66 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2024 Teclib' and contributors.
+ # @copyright 2003-2014 by the INDEPNET Development Team.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+<!-- Button trigger modal -->
+{{ open_btn|raw }}
+
+<!-- Modal -->
+<div class="modal fade" id="{{ modal_id }}" tabindex="-1" role="dialog" aria-hidden="true">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <a type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></a>
+            <div class="modal-status bg-info"></div>
+            <div
+                class="modal-body text-center py-4">
+                <i class="fa-2x ti ti-info-circle mb-2 text-info"></i>
+                <h3>{{ _x('button', 'Update ?') }}</h3>
+                <div class="text-muted">{{ ('A new update is available for the <strong>%plugin_name%</strong> plugin.'|replace({'%plugin_name%': plugin_name}))|raw }}</div>
```

Comment:
1. The message should be translated.
2. The source message should be escaped.
3. The source message should not contains HTML code.
```suggestion
                <div class="text-muted">{{ __('A new update is available for the %s plugin.')|escape|format('<strong>' ~ plugin_name ~ '</strong>')|raw }}</div>
```